tests/installation/partitioning_raid.pm

```diff
@@ -211,52 +210,72 @@ sub run {
-        send_key "shift-tab";
-        send_key "shift-tab";
+        send_key "shift-tab" unless is_storage_ng;
+        send_key "shift-tab" unless is_storage_ng;
 
         # in last step of for loop edit first vda1 and format it as EFI ESP, preparation for fate#322485
         if ($_ eq 'vdd' and get_var('UEFI')) {
             assert_screen 'partitioning_raid-disk_vdd_with_partitions-selected';
-            send_key 'left';                                                                         # fold the drive tree
+            # fold the drive tree
+            send_key 'left';
             assert_screen 'partitioning_raid-hard_disks-unfolded';
-            send_key 'right';                                                                        # select first disk
+            # select first disk
+            send_key 'right';
             assert_screen 'partitioning_raid-disk_vda_with_partitions-selected';
-            send_key 'alt-e';                                                                        # edit first partition
+            # edit first partition
+            send_key 'alt-e';
             assert_screen 'partition-format';
-            send_key 'alt-a';                                                                        # format as FAT (first choice)
+            # format as FAT (first choice)
+            send_key 'alt-a';
             assert_screen 'partitioning_raid-format_fat_UEFI';
-            send_key 'alt-o';                                                                        # mount point selection
+            # mount point selection
+            send_key 'alt-o';
             assert_screen 'partitioning_raid-mount_point-focused';
-            type_string '/boot/efi';                                                                 # enter mount point
+            # enter mount point
+            type_string '/boot/efi';
             assert_screen 'partitioning_raid-mount_point_boot_efi';
             send_key $cmd{finish};
             assert_screen 'expert-partitioner';
             send_key 'shift-tab';
             send_key 'shift-tab';
-            send_key 'left';                                                                         # go to top "Hard Disks" node
+            # go to top "Hard Disks" node
+            send_key 'left';
             assert_screen 'partitioning_raid-hard_disks-unfolded';
-            send_key 'left';                                                                         # fold the drive tree again
+            # fold the drive tree again
+            send_key 'left';
         }
 
         # walk through sub-tree
-        send_key "down";
+        send_key "down" unless is_storage_ng;
     }
 
     # select RAID add
-    assert_screen 'partitioning_raid-raid-selected';
-    send_key $cmd{addraid};
+    if (is_storage_ng) {
+        send_key_until_needlematch 'partitioning_raid-raid-selected', 'down';
+        # hotkey is different on SLE15
+        send_key 'alt-d';
```

Comment:
Let's redefine it like we did for donotformat, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/partitioning.pm#L27